net-irc/polari/polari-46.0.ebuild

```
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="An IRC client for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Polari https://gitlab.gnome.org/GNOME/polari"
```

https://wiki.gnome.org/Apps/Polari is planned for retirement, please use https://apps.gnome.org/Polari instead.
https://gitlab.gnome.org/GNOME/polari/-/blob/46.0/README.md?plain=1#L38
```suggestion
HOMEPAGE="https://apps.gnome.org/Polari https://gitlab.gnome.org/GNOME/polari"
```

---

Done in https://github.com/gentoo/gentoo/commit/7d6cff0bdf173dbed8631c88dcebe0e5cb0236b3